### PR TITLE
PAYARA-1308 email notifier config in admin console added

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

**Comment by Cousjava** [comment]
jenkins test please

**Comment by Cousjava** [comment]
jenkins test please

**Comment by Cousjava** [comment]
jenkins test please

File: appserver/admingui/pom.xml [hunk 1]
```diff
@@ -39,10 +39,7 @@
     only if the new code is made subject to such option by the copyright
     holder.
 
--->
-<!--"Portions Copyright [2016] [Payara Foundation]" -->
-
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```
Comment (Pandrex247) [inline appserver/admingui/pom.xml]: Can you change this back please? The formatting was fine how it was :)

Comment (Cousjava) [inline appserver/admingui/pom.xml]: change made

**Comment by Cousjava** [comment]
jenkins test please

**Pull request merged**